src/server/MessageProcessor.js

```diff
@@ -0,0 +1,419 @@
+/**
+ *  Copyright (c) Facebook, Inc.
```

Comment:
Ok. For reference, we use years in [Relay](https://github.com/facebook/relay/blob/master/src/RelayPublic.js), [graphql-js](https://github.com/graphql/graphql-js/blob/master/src/index.js), and quite a few others, but I am happy to skip it here if our experts say it's ok to do so.